src/icalendar/tests/test_fixed_issues.py

```diff
@@ -482,18 +482,6 @@ def test_issue_237(self):
         self.assertEqual(dtstart.tzinfo.zone, expected_zone)
         self.assertEqual(dtstart.tzname(), expected_tzname)
 
-    def test_issue_345(self):
-        """Issue #345 - Why is tools.UIDGenerator a class (that must be instantiated) instead of a module? """
-        uid1 = icalendar.tools.UIDGenerator.uid()
-        uid2 = icalendar.tools.UIDGenerator.uid('test.test')
-        uid3 = icalendar.tools.UIDGenerator.uid(unique='123')
```

Comment:
The tests are missing these cases:
- [ ] one argument `host_name`
- [ ] one argument `unique`
- [ ] no arguments

Reply:
I create a PR for that...

Reply:
https://github.com/jacadzaca/icalendar/pull/2